pkg/storkctl/applicationrestore.go

```diff
@@ -292,10 +294,22 @@ func waitForApplicationRestore(name, namespace string, ioStreams genericclioptio
 	return msg, err
 }
 
-func getDefaultNamespaceMapping(backup *storkv1.ApplicationBackup) map[string]string {
+func getNamespaceMapping(backup *storkv1.ApplicationBackup, inputMappings string) map[string]string {
+	inputMappingMap := make(map[string]string)
+	if len(inputMappings) > 0 {
+		for _, inputMapping := range strings.Split(inputMappings, ",") {
+			nsMapping := strings.Split(inputMapping, ":")
+			if len(nsMapping) == 2 {
+				inputMappingMap[nsMapping[0]] = nsMapping[1]
+			}
```

Comment:
Do we need to print any errormsg, if the namespace entry was not in the form x:y?

Reply:
Done